criu/cr-service.c

```diff
@@ -339,8 +339,14 @@ static int setup_opts_from_req(int sk, CriuOpts *req)
 	 */
 	if (imgs_changed_by_rpc_conf)
 		strncpy(images_dir_path, opts.imgs_dir, PATH_MAX - 1);
-	else
+	else if (req->has_images_dir_fd)
 		sprintf(images_dir_path, "/proc/%d/fd/%d", ids.pid, req->images_dir_fd);
+	else if (req->images_dir)
```

Comment:
pls explain in the commit message why we need this rpc argument. Why can't you use images_dir_fd?

Reply:
Our RPC client process is in a different pidns and so CRIU can't open its /proc/<pid>/fd/X. For `images_dir_fd` to be useful here it would need to refer to a passed or CRIU's fd.

Reply:
Got it. We can consider sending fd-s in scm.